src/stream/reader/logfile.rs

```diff
@@ -8,13 +8,15 @@ use std::collections::BTreeMap;
 use std::fs::File;
 use std::io::{BufReader, BufWriter, ErrorKind, Read, Seek, SeekFrom, Write};
 use std::path::Path;
+use tako::messages::gateway::CollectedOverview;
 use tako::InstanceId;
 
 pub const HQ_LOG_HEADER: &[u8] = b"HQ:log";
 pub const HQ_LOG_VERSION: u32 = 0;
 
 pub const BLOCK_STREAM_START: u8 = 0;
 pub const BLOCK_STREAM_CHUNK: u8 = 1;
+pub const BLOCK_STREAM_OVERVIEW: u8 = 3;
```

Comment:
It should be named as just BLOCK_OVERVIEW. In log there are saved two things: Streams & Overview.

Reply:
Other then that, the code skeleton seems ok so far.